src/events.rs

```diff
@@ -46,20 +46,28 @@ impl From<EventEntry> for SoonToBeIcsEvent {
 
 #[test]
 fn can_deserialize_event_entry() -> Result<(), serde_json::Error> {
+    use chrono::NaiveDate;
+
     let test: EventEntry = serde_json::from_str(
-        r#"{"Name": "BTI1-TI", "Location": "1060", "Description": "Dozent: HTM", "StartTime": "2022-01-13T11:40:00+01:00", "EndTime": "2022-01-13T12:00:00+01:00"}"#,
+        r#"{"Id": "1", "Name": "BTI1-TI", "Location": "1060", "Description": "Dozent: HTM", "StartTime": "2022-01-13T11:40:00", "EndTime": "2022-01-13T12:00:00"}"#,
```

Comment:
`Id` ist nicht im Event entry, sollte damit auch nicht im Test sein (und der diff wird damit etwas klarer da nur die timezone verschwindet)

```suggestion
        r#"{"Name": "BTI1-TI", "Location": "1060", "Description": "Dozent: HTM", "StartTime": "2022-01-13T11:40:00", "EndTime": "2022-01-13T12:00:00"}"#,
```

Reply:
Stimmt, ich hatte das erst in dem Struct und hab dann gemerkt, dass man das gar nicht braucht